source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -699,6 +699,7 @@ The event API here is assumed to be like the standard `Python Event
             return new ServerDescription
         except (NetworkError, CommandError) as exc:
             close connection
+            conn = None
```

Comment:
`connection = Null` for the reason stated above.

Reply:
Done.